app/src/main/java/me/ccrama/redditslide/Fragments/WikiPage.java

```diff
@@ -70,12 +73,15 @@ public void run() {
     }
 
     private void setUpWebView() {
+        webView.getSettings().setJavaScriptEnabled(true);
+        webView.addJavascriptInterface(new WikiPageJavaScriptInterface(), "Slide");
+
         webView.setWebViewClient(new WebViewClient() {
             @Override
             public boolean shouldOverrideUrlLoading(WebView view, String url) {
                 if (url.toLowerCase().startsWith(wikiUrl.toLowerCase()) && listener != null) {
                     String pagePiece = url.toLowerCase().replace(wikiUrl.toLowerCase(), "")
-                            .split("/")[0]
+                            .split("\\?")[0]
```

Comment:
I will mention that this is now presumptive. I just assume everything after the base Reddit Wiki URL is the page identifier up until any parameters. I don't know if I can do anything better than that.